app/models/custom_value.rb

```diff
@@ -44,6 +56,12 @@ def set_default_value
     end
   end
 
+  # returns the value of this custom value, but converts it according to the field_format
+  # of the custom field beforehand
+  def typed_value
+    strategy.typed_value
+  end
+
   # Returns true if the boolean custom value is true
   def true?
     self.value == '1'
```

Comment:
As in:
````ruby
self.typed_value == true
````

or an explicitly implemented method for all strategies?

I should probably see where this method is used and just remove it...